myhoard/restore_coordinator.py

```diff
@@ -486,6 +486,14 @@ def rebuild_tables(self) -> None:
                 )
                 try:
                     cursor.execute(f"ALTER TABLE {escaped_table_designator} FORCE")
+                except pymysql.err.IntegrityError as e:
+                    # ERROR 1026: Duplicate entry
+                    # This happens for tables that had temporary duplicate rows in different transactions, we can't rebuild
+                    # those at the moment.
+                    if e.args[0] == 1026:
+                        self.log.error("Could not rebuild %s, error: %s, skipping it", escaped_table_designator, str(e))
```

Comment:
Nit: because of we pass it, should it be a warning level?

Reply:
I'd rather have it as an error because we really shouldn't skip tables. Doesn't affect the retention or immediacy of the message.